shared/cfg/guest-os/Linux/Fedora/24.x86_64.cfg

```diff
@@ -0,0 +1,24 @@
+- 24.x86_64:
+    image_name = images/f24-64
+    vm_arch_name = x86_64
+    os_variant = fedora24
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel_params = 'inst.repo=cdrom:/dev/disk/by-label/Fedora-S-24-x86_64'
```

Comment:
Trello cards can get lost since there are many of them, but this code can be copied over and over and till no one actually knows why. I'd prefer if you could include `FIXME` or `BUG` there as well.